vello_encoding/Cargo.toml

```diff
@@ -27,4 +27,4 @@ bytemuck = { workspace = true }
 skrifa = { workspace = true, optional = true }
 peniko = { workspace = true }
 guillotiere = { version = "0.6.2", optional = true }
-smallvec = { workspace = true }
+smallvec = { workspace = true }
```

Comment:
Why have you removed this newline?

Reply:
If i'm being honest, i don't know, i don't even remember lmao, there's no problem if you want that newline back c:

Reply:
There, i *think* that it got back to how it was before this PR